### DISCUSSION TITLE
NetBox maintainers meeting minutes, 09 May 2024

### DISCUSSION THREAD

**Discussion by jeffgdotorg** [netbox-community/netbox#16066] [Announcements]
*NetBox maintainers meeting minutes, 09 May 2024*

_Attending_: Jeremy Stretch, Arthur Hanson, Jeff Gehlbach, Ben Reed, Abhi Saharan, Dan Sheppard

_[Jeff] Need to reschedule the 16 May call, can we push out by 1 hour?_
- *Decision*: Pushing by one hour just for next week.
_[Jeremy] Security reporting_
- Email vs. GitHub: We should pick one
- Originally supported both because GitHub advisories were in beta
- ↳ Dan: Downside to both: Only Jeremy (or another NetBox Labs person) can access reports.
- ↳ Jeremy: Another downside of email is that it lets us ignore things.
- ↳ *Decision*: Stick with GH advisories for now. Currently Jeremy, Arthur, and Jeff have access. Whoever handles intake on a given advisory will do a print-to-PDF and drop the PDF into the Maintainers channel for wider visibility. Jeremy will update SECURITY.txt accordingly.
_[Jeremy] 24 issues tagged with "revisions needed"_
- We need a better system
- Suggest retaining triage ownership until next steps have been identified
- ↳ *Decision*: Immediate step is robo-assigned maintainer keeps assignment until `revisions needed` is cleared. Second step is Jeremy investigates whether the stalebot can help lighten the burden of checking back in by dropping a reminder.
_[Jeff] Better way to spot breaking changes than watching for commits to the release notes?_
- *Decision*: Introduce a `breaking change` label on issues that are expected to result in a change that is either truly breaking (should happen only at major or minor version boundaries) or potentially breaking (e.g. bug fixes that may break integrations that assume the buggy behavior can be relied upon)
_[Jeremy – followup] As we grow, we keep bumping into limitations of GitHub._
- We've talked about using GH Projects to close up some of those gaps.
- ↳ Abhi: In my day job we learned the hard way that GH projects have a limit of 1200 member issues. We moved everything to Jira, but Jira is not a panacea.
- ↳ Jeff: We'll be using GH projects to manage the product backlog and "sprint backlog" starting very soon. Good to know we need to watch out for this pitfall.

**Comment by jeffgdotorg** [comment]
The NetBox Maintainers Meeting scheduled for today (16 May 2024) was canceled by consensus. Two maintainers were unable to attend, and no agenda items had been registered. We expect to meet as scheduled in one week's time.

**Reply by markkuleinio** [reply]
@jeffgdotorg Can I suggest that next time netbox-community maintainers discuss and have a vision about how to get `pynetbox` to support NetBox 4.0? There's PR https://github.com/netbox-community/pynetbox/pull/613 pending that suggests bringing basic 4.0 compatibility.
